### PR TITLE
fix: add search by nft and shorten search placehodler

### LINKED ISSUE #338: Input gets zoomed upon initialization of the page
### Context information

application version: 
device:  Iphone 12
browser: Chrome
operating system: 

### Steps to reproduce

https://github.com/aeternity/aescan/assets/11598532/fbd858f2-d6b7-4f58-b70b-6b91f98c3c03

### LINKED ISSUE #420: Search field placeholder text is too long
### Context information

application version: I can't find it at https://aescan.io/
browser: Google Chrome@116

### Steps to reproduce
- open https://aescan.io/

### What is expected?
To see the whole placeholder text

### What is actually happening?
Screenshot 2023-08-02 at 13 50 46

### LINKED ISSUE #492: Search by NFT

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #420 
fixes #338 
fixes #492 

## Demo

https://github.com/aeternity/aescan/assets/15363559/818e06ef-9e83-46bb-a02e-00a21df3dd69

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] fix nft hints
- [x] add search by nft
- [x] shorten placeholder

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
1. The zoom is still here, but not as much as before. @janmichek , @michele-franchi , is this increase acceptable?

https://github.com/aeternity/aescan/assets/69896204/e1992a12-e5c8-47b2-9b3b-a690a7d4e756

2. Searching is working differently on mobile and web, for example if I'm searching for 'next' on mobile I will get nothing. 

https://github.com/aeternity/aescan/assets/69896204/4705eb48-12fe-4f3b-ba9c-ad36fa1bd847


 The same on web will get me a couple of names with 'next' in naming. 
https://github.com/aeternity/aescan/assets/69896204/cd493745-4f74-43de-a444-d01b0ec0b771

**Comment by janmichek** [comment]
> 1. The zoom is still here, but not as much as before. @janmichek , @michele-franchi , is this increase acceptable?

I tried to implement another fix. Please check it now


> The same on web will get me a couple of names with 'next' in naming. https://github.com/aeternity/aescan/assets/69896204/cd493745-4f74-43de-a444-d01b0ec0b771

It's something I need to request from MDW
https://github.com/aeternity/ae_mdw/issues/1651

**Comment by Liubov-crypto** [comment]
> > 1. The zoom is still here, but not as much as before. @janmichek , @michele-franchi , is this increase acceptable?
> 
> I tried to implement another fix. Please check it now
> 
> > The same on web will get me a couple of names with 'next' in naming. https://github.com/aeternity/aescan/assets/69896204/cd493745-4f74-43de-a444-d01b0ec0b771
> 
> It's something I need to request from MDW [aeternity/ae_mdw#1651](https://github.com/aeternity/ae_mdw/issues/1651)

1. The room on mobile has been fixed. 

2. I see that the search is sensitive to caps lock, for example, if I search for 'helo', it will not find NFT 'HELO'. 

3. I found an additional error during regression, I will register it separatel.

The rest LGTM.

**Comment by janmichek** [comment]
> I see that the search is sensitive to caps lock, for example, if I search for 'helo', it will not find NFT 'HELO'.

Yes that's what I requested to change on MDW side. Applies for all entities in search


> I found an additional error during regression, I will register it separately.

Ok, thanks. Will fix separately


> If you look at the activity of the selected account, the activity tabs, transactions, names, tokens will be unavailable:

I can take a look at this if it was a regression of this work. What is the account hash?

**Comment by Liubov-crypto** [comment]
@janmichek  account hash ak_Cfr7h3XgjD6aScm3A96qMp2EF3p9Ao66gmVQvDEZAAsoCMGER

File: src/stores/search.js [hunk 1]
```diff
@@ -29,10 +30,20 @@ export const useSearchStore = defineStore('search', () => {
     tokensResults.value = data
   }
 
+  async function fetchNftsResults({ query, limit, queryParameters } = {}) {
+    nftsResults.value = null
+    const defaultParameters = `/v2/aex141?prefix=${query}&limit=${limit ?? 10}&direction=forward`
+    const { data } = await axios.get(`${MIDDLEWARE_URL}${queryParameters || defaultParameters}`)
+    console.log('data', data)
```
Comment (michele-franchi) [inline src/stores/search.js:37]:
```suggestion
```

File: src/components/SearchNftsTableCondensed.vue [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            </app-tooltip>
+          </th>
+          <td class="search-nfts-table-condensed__data">
+            <app-link
+              :to="`/nfts/${nft.contractId}`"
+              class="search-nfts-table-condensed__link">
+              {{ nft.symbol }}
+            </app-link>
+          </td>
+        </tr>
```
Comment (michele-franchi) [inline src/components/SearchNftsTableCondensed.vue:24]:
```suggestion
```

I would remove the "Symbol" column to keep it consistent with the NFTs page where we don't have it.

File: src/components/SearchNftsTable.vue [hunk 3]
```diff
@@ -0,0 +1,75 @@
+<template>
+  <table class="search-nfts-table">
+    <thead>
+      <tr>
+        <th>
+          Symbol
+          <hint-tooltip>
+            {{ nftsHints.symbol }}
+          </hint-tooltip>
+        </th>
```
Comment (michele-franchi) [inline src/components/SearchNftsTable.vue:10]:
```suggestion
```

I would remove the "Symbol" column to keep it consistent with the NFTs page where we don't have it.

File: src/components/SearchNftsTable.vue [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      <tr
+        v-for="nft in nfts.data"
+        :key="nft.contractId">
+        <td>
+          <app-link
+            :to="`/nfts/${nft.contractId}`"
+            class="search-nfts-table__link">
+            {{ nft.symbol }}
+          </app-link>
+        </td>
```
Comment (michele-franchi) [inline src/components/SearchNftsTable.vue:35]:
```suggestion
```

File: src/components/TheSearchBar.vue [hunk 5]
```diff
@@ -131,7 +131,7 @@ function isMicroblockId(query) {
     border: none;
     background-color: var(--color-white);
     margin-right: var(--space-1);
-    font-size: 14px;
+    font-size: 16px;
```
Comment (michele-franchi) [inline src/components/TheSearchBar.vue:134]: Very small detail: I would leave 14px for desktop and set 16px only for mobile devices.

File: src/pages/search/[id].vue [hunk 6]
```diff
@@ -23,11 +26,13 @@
 import PageHeader from '@/components/PageHeader'
 import AppTabs from '@/components/AppTabs'
 import AppTab from '@/components/AppTab'
+import SearchNftPanel from '~/components/SearchNftsPanel'
+// todo fix imports
```
Comment (michele-franchi) [inline src/pages/search/[id].vue:30]: Is this solved?

File: src/utils/hints/nftsHints.js [hunk 7]
```diff
@@ -1,7 +1,23 @@
 export const nftsHints = {
   nft: 'A non-fungible token (NFT) is a unit of data stored on a blockchain, that certifies a digital asset to be unique and therefore not interchangeable. NFTs can be used to represent items such as photos, videos, audio, and other types of digital files.',
+  symbol: 'Non-unique symbol of the NFT.',
```
Comment (michele-franchi) [inline src/utils/hints/nftsHints.js:3]:
```suggestion
```

**Review by michele-franchi — CHANGES_REQUESTED** [review]
Please have a look at my comments.

File: src/components/TheSearchBar.vue (hunk 5 above)
Comment (janmichek) [reply src/components/TheSearchBar.vue:134]: good idea, adjusted

File: src/pages/search/[id].vue (hunk 6 above)
Comment (janmichek) [reply src/pages/search/[id].vue]: haha, now yes.

File: src/components/SearchNftsTable.vue (hunk 4 above)
Comment (janmichek) [reply src/components/SearchNftsTable.vue]: removed

**Comment by janmichek** [comment]
> @janmichek account hash ak_Cfr7h3XgjD6aScm3A96qMp2EF3p9Ao66gmVQvDEZAAsoCMGER

Fixed by rebase

**Review by Liubov-crypto — APPROVED** [review]
Fixed, LGTM

**Review by michele-franchi — APPROVED** [review]
Good job

**Pull request merged**